modules/accredited_representative_portal/app/policies/accredited_representative_portal/application_policy.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class ApplicationPolicy
+    attr_reader :user, :record
+
+    def initialize(user, record)
+      @user = user
+      @record = record
+    end
+
+    def index?
+      override_warning
+      false
+    end
+
+    def show?
+      override_warning
+      false
+    end
+
+    def create?
+      override_warning
+      false
+    end
+
+    def new?
+      create?
+    end
+
+    def update?
+      override_warning
+      false
+    end
+
+    def edit?
+      update?
+    end
+
+    def destroy?
+      override_warning
+      false
+    end
+
+    private
+
+    def override_warning
```

Comment:
@nihil2501 Oren, I added this just as a reminder to override the base methods. I was thinking of doing a hard exception but I thought softer would be ok since the actual template from Pundit didn't even have anything. 

Or maybe it's not necessary at all. What do you think?

Reply:
@ojbucao Neat. This does seem like useful DX. The scenario is that a dev feels that a particular resource should be authorized in some controller action and if so there surely should be some logic corresponding to that rather than always returning false. I like it.